File: snippet/query-examples/ocaml/har-get.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "https://will-x86.com/api/pixelsq?x=1&y=2&hex=%23102932" in
let headers = Header.add_list (Header.init ()) [
  ("User-Agent", "curl/7.68.0");
  ("Accept", "*/*");
] in

Client.call ~headers `GET uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)

File: snippet/query-examples/c/har-get.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "GET");
curl_easy_setopt(hnd, CURLOPT_URL, "https://will-x86.com/api/pixelsq?x=1&y=2&hex=%23102932");

struct curl_slist *headers = NULL;
headers = curl_slist_append(headers, "User-Agent: curl/7.68.0");
headers = curl_slist_append(headers, "Accept: */*");
curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);

CURLcode ret = curl_easy_perform(hnd);
